src/ucp/proto/proto_select.c

```diff
@@ -464,11 +486,11 @@ static ucs_status_t ucp_proto_select_elem_add_envelope(
 
 
 static ucs_status_t
-ucp_proto_select_elem_init_thresh(ucp_worker_h worker,
-                                  ucp_proto_select_elem_t *select_elem,
+ucp_proto_select_elem_init_thresh(ucp_proto_select_elem_t *select_elem,
                                   ucp_proto_select_init_protocols_t *proto_init,
                                   ucp_worker_cfg_index_t ep_cfg_index,
-                                  ucp_worker_cfg_index_t rkey_cfg_index)
+                                  ucp_worker_cfg_index_t rkey_cfg_index,
+                                  unsigned window_size)
```

Comment:
do we really need this param? I guess it will always be taken from the config, so maybe just take it from there in `ucp_proto_thresholds_next_range`?